zerver/views/home.py

```diff
@@ -179,7 +179,7 @@ def home_real(request):
         server_inline_image_preview = settings.INLINE_IMAGE_PREVIEW,
         server_inline_url_embed_preview = settings.INLINE_URL_EMBED_PREVIEW,
         password_min_length = settings.PASSWORD_MIN_LENGTH,
-        password_min_quality = settings.PASSWORD_MIN_ZXCVBN_QUALITY,
+        password_min_quality = settings.PASSWORD_MIN_ENTROPY_BITS,
```

Comment:
This parameter's name should change to match the setting -- `password_min_entropy_bits`.

Ditto for the similar code above.